inc/mqttlog.class.php

```diff
@@ -48,6 +53,11 @@ public function __construct() {
 
    }
 
+   public function getRights($interface = 'central') {
+      $rights = parent::getRights();
+      return $rights;
+   }
```

Comment:
if you don't need to add a specific right, then you can simply let the implementation in commondbtm do the job.